rafx-framework/src/nodes/frame_packet.rs

```diff
@@ -179,22 +185,33 @@ impl FramePacketBuilder {
     pub fn add_view(
```

Comment:
we may want to reconsider the name for this function since now it's doing the full visibility check, (assuming I understand the code correctly.) Or make this function take results and leave it to the user to call the query function. This would make it more clear to a user that it's potentially a very expensive operation.

Reply:
query_visibility_and_add_results is wordy but would make it clear what this function is doing and that it's a significant operation

Reply:
I'd like to punt on changing the name because I imagine that this function will need to be modified very soon -- once we expose callbacks for the user to do their own threading, we'll have to yank the visibility code out of here. That was something I was going to look at as part of this work, but as the lines of code kept piling up I felt it was more important to ship a first increment and then iterate on it.

Reply:
We could temporarily rename it to query_visibility_and_add_results for now (since it's an easy/quick change) and let that motivate us to find something better later.

Reply:
(I do agree with the sentiment that we shouldn't pile on more things to an already-large PR, and this is a large enough feature that it will need to be iterated on)